src/main/java/core/basesyntax/Demo.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Demo {
+    public static void main(String[] args) {
```

Comment:
we should use main() method only locally for testing, so when you create a PR better remove this method.

Reply:
done